frontend/src/utils/server-utils.ts

```diff
@@ -546,7 +565,7 @@ export async function getLayersAvailableDates(
   const layersWithValidity: ValidityLayer[] = Object.values(LayerDefinitions)
     .filter(layer => layer.validity !== undefined)
     .map(layer => {
-      const layerId = layer.type === 'wms' ? layer.serverLayerName : layer.id;
+      const layerId = layer.id;
```

Comment:
nice catch here, i see that `mapServerDatesToLayerIds` always returns `layer.id` and not `layer.serverLayerName`